task/pa11y.js

```diff
@@ -16,59 +16,63 @@
 
 const async = require('async');
 const chalk = require('chalk');
-const CronJob = require('cron').CronJob;
+const {CronJob} = require('cron');
 
 module.exports = initTask;
 exports.runPa11yOnTasks = runPa11yOnTasks;
 
 // Initialise the task
 function initTask(config, app) {
+
```

Comment:
I don't know if the linter enforces this, but we don't usually add an empty line after an open bracket?